src/styles/main.css

```diff
@@ -1,3 +1,5 @@
+/* stylelint-disable csstree/validator */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
 * {
   padding: 0;
   margin: 0;
```

Comment:
Kindly, stick to the design provided as much as possible. Use the appropriate `font-family` to match your design to the given design.

| Your Design | Figma Design |
|-------------|---------------|
| <img width="671" alt="image" src="https://user-images.githubusercontent.com/34578631/173042168-5961f092-97a7-46ec-9626-de06c0db4aa9.png"> | <img width="595" alt="image" src="https://user-images.githubusercontent.com/34578631/173042222-cbc21026-e0ad-433e-a747-f546f2778985.png"> |

Also, give the proper background color as of the design only the `nav bar` and the box with the detail of book have white background everything else have its background color. Please add separator between the body and `nav bar`.

Reply:
@Mre55 Thanks for providing me with the feedback, I am going to implement them.